### PR TITLE
Add simple test

### PR DESCRIPTION
**Pull request by eye-dee**

### CODE REVIEW

**Comment by pepya4ka** [comment]
Thx

File: src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
     @MessageMapping("/main.start")
     @SendTo("/game")
     public GameState start() {
-        return new GameState();
+        return new GameState(1);
```
Comment (romangr) [inline src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java:15]: Magic number should be extracted to a constant

Comment (pepya4ka) [reply src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java:15]: are u on main branch?

Comment (eye-dee) [reply src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java:15]: Ty chto tut pisheh?

Comment (pepya4ka) [reply src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java:15]: kto?

Comment (eye-dee) [reply src/main/java/com/pepyachka/myspringbootwebsocketexample/controller/MainController.java:15]: Roma

**Pull request merged**